demo/admin/src/generator/utils/writeGenerated.ts

```diff
@@ -0,0 +1,22 @@
+import { ESLint } from "eslint";
+import { promises as fs } from "fs";
+import * as path from "path";
+
+export async function writeGenerated(filePath: string, contents: string): Promise<void> {
```

Comment:
I skipped this for now in my implementation, but I already planned this could be loosely based on a generic component-file structure...

- es-lint (hopefully not needed, but probably needed if correct formatting is not so easy...)
- imports
- preparations (e.g. graphql queries, if they are not placed in a separate file)
- component-init (the code after `export function MyComponent(...) {`
- render-check (stuff like error || loading => render progress)
- render-return

delegating the content-generation for the different sections to something not completely thought-out. (my first idea was defining form and grid with an element-types, configured/initialized with the developer-written form/field config.)